docs/querying/lookups.md

```diff
@@ -55,21 +56,37 @@ Other lookup types are available as extensions, including:
 Query Syntax
 ------------
 
-In [Druid SQL](sql.html), lookups can be queried using the `LOOKUP` function, for example:
+In [Druid SQL](sql.html), lookups can be queried using the [`LOOKUP` function](sql.md#string-functions), for example:
 
+```sql
+SELECT
+  LOOKUP(store, 'store_to_country') AS country,
+  SUM(revenue)
+FROM sales
+GROUP BY 1
 ```
-SELECT LOOKUP(column_name, 'lookup-name'), COUNT(*) FROM datasource GROUP BY 1
+
+They can also be queried using the [JOIN operator](datasource.md#join):
```

Comment:
Do we want to talk about direct querying that was added in #9502

Reply:
I don't think so, because the assumption should be you can direct query it (why not!)

Reply:
^ fair point, this documentation makes me think there are only 2 ways to query a lookup, but maybe it's because I'm biased and have not been able to query it before that PR :)